AnkiDroid/src/main/java/com/ichi2/anki/widgets/DeckAdapter.java

```diff
@@ -198,13 +206,68 @@ public void onBindViewHolder(ViewHolder holder, int position) {
+        boolean showRemainingCards;
+        boolean showMinimumInterval;
+        try {
+            showRemainingCards = mCol.getConf().getBoolean("dueCounts");
+            // why isn't what follows another call to mCol.getConf()...?
+            // didn't want to modify anki collection prefs json object at this point
+            // so it must go through Android's preference system, which requires a Context object
+            Context context = holder.itemView.getContext();
+            showMinimumInterval = AnkiDroidApp.getSharedPrefs(context).getBoolean(context.getString(R.string.pref_show_min_interval), false);
+        } catch (JSONException e) {
+            throw new IllegalStateException("Failed to find 'dueCounts' field in collection prefs");
+        }
+
         // Set the card counts and their colors
-        holder.deckNew.setText(String.valueOf(node.newCount));
-        holder.deckNew.setTextColor((node.newCount == 0) ? mZeroCountColor : mNewCountColor);
-        holder.deckLearn.setText(String.valueOf(node.lrnCount));
-        holder.deckLearn.setTextColor((node.lrnCount == 0) ? mZeroCountColor : mLearnCountColor);
-        holder.deckRev.setText(String.valueOf(node.revCount));
-        holder.deckRev.setTextColor((node.revCount == 0) ? mZeroCountColor : mReviewCountColor);
+        if (showRemainingCards) {
+            holder.countsLayout.setVisibility(View.VISIBLE);
+            holder.deckNew.setVisibility(View.VISIBLE);
+            holder.deckLearn.setVisibility(View.VISIBLE);
+            holder.deckRev.setVisibility(View.VISIBLE);
+            holder.deckNew.setText(String.valueOf(node.newCount));
+            holder.deckNew.setTextColor((node.newCount == 0) ? mZeroCountColor : mNewCountColor);
+            holder.deckLearn.setText(String.valueOf(node.lrnCount));
+            holder.deckLearn
+                .setTextColor((node.lrnCount == 0) ? mZeroCountColor : mLearnCountColor);
+            holder.deckRev.setText(String.valueOf(node.revCount));
+            holder.deckRev.setTextColor((node.revCount == 0) ? mZeroCountColor : mReviewCountColor);
+        } else {
+            // ViewHolder objects are (potentially) reused, so it's important to
+            // always be explicity about the state that you want. It's not enough to just have
+            // the above "if" put the views in the proper state, we need this
+            // else as well
+            holder.countsLayout.setVisibility(View.INVISIBLE);
+            holder.deckNew.setVisibility(View.INVISIBLE);
+            holder.deckLearn.setVisibility(View.INVISIBLE);
+            holder.deckRev.setVisibility(View.INVISIBLE);
+        }
+
+        // Show/hide the minimum interval information
+        if (showMinimumInterval) {
+            holder.minIvlLayout.setVisibility(View.VISIBLE);
+            holder.minIvl.setVisibility(View.VISIBLE);
+            int doneReviewingIvl = 14; // if minIvl is 14 days, you are g2g
```

Comment:
Can we change this to 21, to match [Anki's definition](https://apps.ankiweb.net/docs/manual.html#types-of-cards) of 'mature'? I know that's an arbitrary # too but I would rather standardize.

Reply:
done